src/modules/robot/arm_solutions/MorganSCARASolution.cpp

```diff
@@ -58,7 +60,7 @@ float MorganSCARASolution::to_degrees(float radians) const
 
 void MorganSCARASolution::cartesian_to_actuator(const float cartesian_mm[], ActuatorCoordinates &actuator_mm ) const
 {
-
+  if (!THEROBOT->disable_arm_solution){
```

Comment:
This is wrong. there are no changes to the arm solution. when disable-arm-solution is set the arm solution is never called by robot.